common/src/main/java/com/redhat/cloud/notifications/openbridge/BridgeEventService.java

```diff
@@ -0,0 +1,25 @@
+package com.redhat.cloud.notifications.openbridge;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.Map;
+
+/**
+ * Deliver events to the OpenBridge ingress
+ */
+@Path("/events")
+public interface BridgeEventService {
+
+    @POST
+    @Path("/")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
That's not what I understand from [the doc](https://github.com/eclipse/microprofile-rest-client/blob/master/spec/src/main/asciidoc/clientexamples.asciidoc#sample-definitions):
>Users may specify the media (MIME) type of the outbound request using the @Consumes annotation (this determine the Content-Type HTTP header), and the expected media type(s) of the response by using the @Produces annotation (the Accept HTTP header). This indicates that the client interface expects that the remote service consumes/produces the specified types. If no @Consumes or @Produces annotation is specified for a given request, it will default to jakarta.ws.rs.core.MediaType.APPLICATION_JSON ("application/json").